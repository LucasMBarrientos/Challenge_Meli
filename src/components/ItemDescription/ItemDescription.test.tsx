import React from "react";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom";

import { ItemDescription,ItemDescriptionProps } from "./";

const defaultProps: ItemDescriptionProps = {
  id: "test_id",
  text: "test_text",
  price: 0,
  ubicacion: "test_location",
  currency_id: "test_currencyId",
};


describe("<Header />", () => {
  it("Should render default without crash", () => {
    const { asFragment } = render(<ItemDescription {...defaultProps}/>);

    expect(asFragment()).toMatchSnapshot();
  });

});
