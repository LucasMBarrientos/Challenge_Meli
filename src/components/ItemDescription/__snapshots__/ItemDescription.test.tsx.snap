// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Should render default without crash 1`] = `
<DocumentFragment>
  <div
    class="MuiGrid-root MuiGrid-container root css-11lq3yg-MuiGrid-root"
  >
    <div
      class="MuiGrid-root MuiGrid-item itemImage css-13i4rnv-MuiGrid-root"
    >
      <img
        height="200"
        width="200"
      />
    </div>
    <div
      class="MuiGrid-root MuiGrid-item itemDescriptionContainer css-13i4rnv-MuiGrid-root"
    >
      <p
        class="MuiTypography-root MuiTypography-body1 itemPrice css-ahj2mt-MuiTypography-root"
      >
        $ 0
      </p>
      <div
        class="itemDescription MuiBox-root css-0"
      >
        test_text
      </div>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item itemUbicacion css-13i4rnv-MuiGrid-root"
    >
      test_location
    </div>
  </div>
</DocumentFragment>
`;
